tasks/selinux_load_module.yml

```
@@ -14,57 +14,40 @@
         src: "{{ item.path }}"
         mode: preserve
 
-    # yamllint disable rule:line-length
-    - name: Install module with priority
-      command: >-
-        semodule -n -s {{ store }} -X {{ priority }} -i {{ tempdir.path }}/{{ item.path | basename }}
-      when: priority != "0" and selinux_priorities
+    - name: Install the module
+      local_semodule:
+        path: "{{ tempdir.path }}/{{ item.path | basename }}"
+        priority: "{{ priority }}"
+        state: "enabled"
```

```suggestion
        state: enabled
```
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax
```
Do not use quotes unless you have to, especially for short module-keyword-like strings like present, absent, etc. But do use quotes for user-side strings such as descriptions, names, and messages.
```
I realize there are other places in the selinux yaml files where quotes are used, so perhaps not using quotes in this case makes this not consistent with other places in the file, but it is preferred not to use quotes in this case for new code.